packages/thegraph-data-access/test/index.test.ts

```diff
@@ -0,0 +1,5 @@
+describe('the test', () => {
+  it('should pass', () => {
+    expect(true).toBe(true);
+  });
+});
```

Comment:
```suggestion
```

Reply:
That will break the learn as it finds a test script in the package.json. I will remove the test script instead then

Reply:
Yes that's what I meant too 👌